### PR TITLE
Add a note to the readme about race condition

### PR DESCRIPTION
**Pull request by nagen010**
I have updated the README.md with the port availability limitation. Please review and let me know if I need to elaborate the limitation.

### CODE REVIEW

**Comment by sindresorhus** [comment]
> Please review and let me know if I need to elaborate the limitation.

Yes, please :)

**Comment by nagen010** [comment]
@sindresorhus Please review and let me know if it is okay :)

File: readme.md [hunk 1]
```diff
@@ -29,7 +29,8 @@ getPort({port: 3000}).then(port => {
 	// Will use 3000 if available, otherwise fall back to a random port
 });
 ```
-
+## Limitations
+-  In practice, under typical circumstances, the port number almost always will be available to use. But, there's a possibility of race condition as another service may have started to occupy the port in between the time it takes to determine whether the port is available and the program actually starts using the port. So, It isn't always guaranteed to be available.
```
Comment (sindresorhus) [inline readme.md]: Yes, that's the info I wanted, but I think it could be written more succinctly.

Comment (nagen010) [reply readme.md]: Sure, I will make it more clear.

**Pull request merged**